build.gradle

```diff
@@ -982,7 +987,19 @@ project('standalone') {
             classpath += files(systemProperties.get("extDirs"))
         }
 
-        jvmArgs "-Xdebug", "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=n"
+        def JDWP_DEFAULT_PORT = 8050
+        def jvmDebugPort = System.getenv("JDWP_PORT")
+        if (jvmDebugPort != null && jvmDebugPort != "") {
```

Comment:
this outer `if` should not be required since we are using regex match at line no `993`. This can be removed.

Reply:
If we did not check for null then initial scenario where JDWP port is not given by user,  jvmDebugPort value line no 991 gives null and according regex this value does not match and we get below line in startup log. So added this check before validating through regex.

"Invalid JDWP_PORT number - null, staring Pravega standalone using default port 8050"